### PR TITLE
feat: add extra volumes mount

### PR DESCRIPTION
**Pull request by DanilaKazakevich**
Allows to customize Nginx or PHP parameters by mounting additional volumes. 
My changes for adding extraVolumes based on [bitnami wordpress chart](https://github.com/bitnami/charts/tree/master/bitnami/wordpress)

### CODE REVIEW

**Comment by nxf5025** [comment]
@bdashrad - Any idea when this can get reviewed/released?

File: charts/privatebin/templates/_helpers.tpl [hunk 1]
```diff
@@ -52,3 +52,16 @@ Return the appropriate apiVersion for podsecuritypolicy.
...
+Usage:
+{{ include "privatebin.tplvalues.render" ( dict "value" .Values.path.to.the.Value "context" $) }}
+*/}}
+{{- define "privatebin.tplvalues.render" -}}
+    {{- if typeIs "string" .value }}
+        {{- tpl .value .context }}
+    {{- else }}
+        {{- tpl (.value | toYaml) .context }}
+    {{- end }}
+{{- end -}}
```
Comment (bdashrad) [inline charts/privatebin/templates/_helpers.tpl:67]:
```suggestion
{{- end -}}

```

**Review by bdashrad — APPROVED** [review]
Just needs to bump chart version.

**Comment by bdashrad** [comment]
Closed in favor of #81

**Pull request closed without merging**